### PR TITLE
Update tap basic tutorial doc for 1488 and 1489 support

### PR DESCRIPTION
**Pull request by jon-perreault**
### What does this Pull Request accomplish?

Update the basic tap tutorial document for 1488 and 1489 hardware support.

### CODE REVIEW

File: docs/tutorials/gettingstartedexample/gse-tap-basic.md [hunk 1]
```diff
@@ -15,20 +15,28 @@ This tutorial will teach you the steps needed to configure a tap acquisition usi
...
+- [PXIe-1487 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1487-getting-started/)
+- [PXIe-1488 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1488-getting-started/)
+- [PXIe-1489 Getting Started Guide](https://www.ni.com/docs/en-US/bundle/pxie-1489-getting-started/)
 
 Basic knowledge of LabVIEW and LabVIEW FPGA concepts. 
 
 A camera supported by the getting started example configuration scripts (i.e. Leopard Imaging IMX490).
 
-A supported interface module that includes both serial input and serial output channels (tap module) and a second supported interface module that includes serial input channels (acq module) on a PXI system running Windows.
+A supported interface module that includes both serial input and serial output channels (tap module) and a second supported interface module that includes serial input channels (acq module) on a PXI system running Windows. The table below lists all supported tap and acq modules and supported cameras that have configuration scripts provided with the getting started example. Configuration scripts for cameras not listed in the table below need to be provided separately.
```
Comment (hernstrom) [inline docs/tutorials/gettingstartedexample/gse-tap-basic.md]: Suggest rewording for clarity at a minimum, but see the further comment below for a more comprehensive potential change:

"For cameras not listed in the table below, configuration scripts need to be provided separately."

File: docs/tutorials/gettingstartedexample/gse-tap-basic.md [hunk 2]
```diff
@@ -15,20 +15,28 @@ This tutorial will teach you the steps needed to configure a tap acquisition usi
...
+| PXIe-1487 (8 In - 9296A Deserializer)        | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes)  | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1488 (8 In - 9702 Deserializer)         |                                                                                                                                       |
+| PXIe-1488 (4 In 4 Out - 971/9702 SerDes)     |                                                                                                                                       |
+| PXIe-1489 (4 In - 96792A Deserializer)       |                                                                                                                                       |
+| PXIe-1489 (2 In 2 Out - 96793/96792A SerDes) |                                                                                                                                       |
```
Comment (hernstrom) [inline docs/tutorials/gettingstartedexample/gse-tap-basic.md]: (Same comment as from the acq doc update)
Why are we listing blank rows in the table? Is it just to convey that this module doesn't have a script that is provided?

If scripts are coming soon, then blank rows seems ok, but if nota nother option, remove the blank rows and change the last sentence above the table to read:

"For modules & cameras not listed in the table below, configuration scripts need to be provided separately."

File: docs/tutorials/gettingstartedexample/gse-tap-basic.md (hunk 1 above)
Comment (jon-perreault) [reply docs/tutorials/gettingstartedexample/gse-tap-basic.md]: I made the suggested change.

File: docs/tutorials/gettingstartedexample/gse-tap-basic.md (hunk 2 above)
Comment (jon-perreault) [reply docs/tutorials/gettingstartedexample/gse-tap-basic.md]: I removed the rows from the table for modules that do not have camera configuration scripts provided with the getting started example.

**Review by hernstrom — APPROVED** [review]
